manual/C/ch_Finance-Quote.xml

```diff
@@ -240,6 +251,199 @@ SYNOPSIS
+"fidelityfixed" "financecanada" "finanzpartner" "finland" "fool" "france" "ftfunds" "ftportfolios"
+"ftportfolios_direct" "fundlibrary" "goldmoney" "greece" "hex" "hu" "hufund" "hungary" "hustock"
+"indiamutual" "known_currencies" "lerevenu" "maninv" "morningstar" "morningstarjp" "mstaruk" "nasdaq"
+"nyse" "nz" "nzx" "platinum" "romania" "seb_funds" "sixfunds" "sixshares" "stockhousecanada_fund"
+"tdefunds" "tdwaterhouse" "tiaacref" "tnetuk" "troweprice" "troweprice_direct" "trustnet" "tsp" "tsx"
+"uk_unit_trusts" "ukfunds" "unionfunds" "usa" "usfedbonds" "vanguard" "vwd" "yahoo" "yahoo_asia"
+"yahoo_australia" "yahoo_brasil" "yahoo_europe" "yahoo_json" "yahoo_nz" "yahoo_yql" "za" "za_unittrusts")
+</screen>
+        </para>
+      </formalpara>
+
+      <para>The first part is the error message about a missing &app-perl;-modul Mozilla/CA.pm. At the end in
+        parentheses is the normal output of the currently installed &app-fq; version<footnote>
+        <para>The most recent &app-fq; version is &app-fq-vers;.</para>
+        </footnote>
+        and a list of available sources.
+      </para>
+
+      <tip>
+        <para>If an error is displayed, see the next section <xref linkend="gnc-fq-update" />.
+          Otherwise, continue with <xref linkend="gnc-fq-dump" />.
+        </para>
+      </tip>
+    </sect2>
+
+    <sect2 id="gnc-fq-update">
+      <title>gnc-fq-update</title>
+
+      <abstract>
+        <para>This program installs or updates the &app-fq; software module along with its dependencies.
+          <note>
+            <para>Windows users are generally better off using the update-tool &mc.start.win-install-fq;.
+            </para>
+          </note>
+        </para>
+      </abstract>
+
+      <note>
+        <para>This program needs superuser or administrative privileges to succeed in &lin; or &mac; but not in &win;.
+        </para>
+      </note>
+    </sect2>
+
+    <sect2 id="gnc-fq-dump">
+      <title>gnc-fq-dump</title>
+<!-- Todo: insert link to wiki and/or guide -->
+      <abstract>
+        <para>This program returns quote data for a source and a list of symbols in a
+          format which is easy to read for humans. It is useful for checking that a
+          given online quote source is up and functional. 
+        </para>
+      </abstract>
+
+      <para>You can use this command to verify that the symbol <!-- <xref linkend="tool-ge-symbol" /> -->,
+        which you want to use for your security for online quote retrieval, works for the
+        desired qoute source. <!-- <xref linkend="tool-ge-TypeQuoteSource" /> -->.
+      </para>
+      
+      <tip>
+        <para>With <userinput>gnc-fq-dump</userinput> you can check symbols faster than from &app;
+          if an error occurs during the retrieval with &app;, because the output of the script
+          is better suited for analysis and not all working symbols are retrieved each time,
```

Comment:
"output of the script is better suited for analysis" is dubious at best.  "Might save you from running GnuCash with debug logging enabled" would be more accurate.

Reply:
BTW, the command line parameters are not described in the help yet.